Makefile

```diff
@@ -1,5 +1,5 @@
 TESTDIR=tmp-test-dir-stravalib
-PYTEST_ARGS=--import-mode=importlib --cov stravalib stravalib/tests/unit stravalib/tests/integration
+PYTEST_ARGS=--cov stravalib stravalib/tests/unit stravalib/tests/integration
 
 ## I still need to add code cov to this build
```

Comment:
```suggestion
```
Since we added code cov - i keep seeing my little TODO comments throughout our code base :)